packages/sdk/src/grpc/translation.ts

```diff
@@ -339,33 +339,60 @@ export function accountInfo(acc: v2.AccountInfo): v1.AccountInfo {
     const aggAmount = acc.encryptedBalance?.aggregatedAmount?.value;
     const numAggregated = acc.encryptedBalance?.numAggregated;
 
-    const encryptedAmount: v1.AccountEncryptedAmount = {
+    const accountEncryptedAmount: v1.AccountEncryptedAmount = {
         selfAmount: unwrapValToHex(acc.encryptedBalance?.selfAmount),
         startIndex: unwrap(acc.encryptedBalance?.startIndex),
         incomingAmounts: unwrap(acc.encryptedBalance?.incomingAmounts).map(unwrapValToHex),
         // Set the following values if they are not undefined
         ...(numAggregated && { numAggregated: numAggregated }),
         ...(aggAmount && { aggregatedAmount: unwrapToHex(aggAmount) }),
     };
-    const releaseSchedule = {
+    const accountReleaseSchedule = {
         total: CcdAmount.fromProto(unwrap(acc.schedule?.total)),
         schedule: unwrap(acc.schedule?.schedules).map(trRelease),
     };
-    const cooldowns = acc.cooldowns.map(transCooldown);
-    const availableBalance = CcdAmount.fromProto(unwrap(acc.availableBalance));
+    const accountCooldowns = acc.cooldowns.map(transCooldown);
+    const accountAmount = CcdAmount.fromProto(unwrap(acc.amount));
+
+    let accountAvailableBalance: CcdAmount.Type;
+    if (acc.availableBalance !== undefined) {
```

Comment:
**Suggestion:**
It would be good to add a comment on why this can be undefined, since the current gRPC makes it looks like it cannot, so we should add this context.
It should include what node versions this will actually be relevant for

Reply:
Good suggestion, will add this